application/src/main/resources/db/V2_Add_Warn_System.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE warns
+(
+    userid      TEXT NOT NULL PRIMARY KEY,
+    guildid     TEXT NOT NULL,
+    timeswarned TEXT NOT NULL
```

Comment:
conflicts naming convention, which is lower_snake_case
* `user_id`
* `guild_id`
* `times_warned`

Reply:
`timeswarned` is a bad name anyways, maybe `warnings` or `warning_count`  or `warning_amount`